### PR TITLE
Zonefstests

### PR DESCRIPTION
**Pull request by morbidrsa**
Add zonefs testing to rapido.

### CODE REVIEW

File: cut/zonefstests_nullblk.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+RAPIDO_DIR="$(realpath -e ${0%/*})/.."
+. "${RAPIDO_DIR}/runtime.vars"
+
+_rt_require_dracut_args
+
+"$DRACUT" \
+	--install "ps rmdir dd id basename stat wc grep blkzone cut fio \
+		   rm truncate" \
+	--include "$ZONEFSTOOLS_SRC/src/mkzonefs" "/usr/bin/mkzonefs" \
```
Comment (ddiss) [inline cut/zonefstests_nullblk.sh]: does `mkzonefs` have any shared object dependencies? If so, I'd prefer to add it to the `--install` list and add `$ZONEFSTOOLS_SRC/src` to `PATH` in the autorun file, like we do for fstests.

File: cut/zonefstests_nullblk.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#
+# This library is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public
+# License for more details.
+
+RAPIDO_DIR="$(realpath -e ${0%/*})/.."
+. "${RAPIDO_DIR}/runtime.vars"
+
+_rt_require_dracut_args
```
Comment (ddiss) [inline cut/zonefstests_nullblk.sh]: Please add `_rt_require_conf_dir ZONEFSTOOLS_SRC` here

File: cut/zonefstests_nullblk.sh (hunk 1 above)
Comment (morbidrsa) [reply cut/zonefstests_nullblk.sh]: It actually does have libuiid.so and libblkid.so, but I don't think mkzonefs is packaged for any distribution yet.

I'll look at how we're resolving this in fstests.

File: cut/zonefstests_nullblk.sh (hunk 2 above)
Comment (morbidrsa) [reply cut/zonefstests_nullblk.sh]: done

File: cut/zonefstests_nullblk.sh (hunk 1 above)
Comment (morbidrsa) [reply cut/zonefstests_nullblk.sh]: done

**Comment by ddiss** [comment]
Merged - thanks!

**Pull request merged**